instance/models/openedx_instance.py

```diff
@@ -392,3 +396,63 @@ def delete(self, *args, **kwargs):
         self.deprovision_s3()
         self.deprovision_rabbitmq()
         super().delete(*args, **kwargs)
+
+    def update_consul_metadata(self):
+        """
+        This method is going over all fields in this model to reflect
+        their values on Consul.
+        Values on consul will be fetched first, then updated only if
+        they changed in this version of the model using agent's `cas`
+        parameter (Check-And-Set). If PUT operation successfully
+        updated the field then we update the configuration version
+        number in Consul.
+
+        :note: This still doesn't apply removed-configurations case.
+
+        :return: The current version number, and the if they include
+                 changes or not.
+        """
+        if not settings.CONSUL_ENABLED:
+            return 0, False
+
+        agent = consul.Consul()
+        active_servers_num = len(self.get_active_appservers())
```

Comment:
Good catch 👍. I also just noticed that `get_active_appservers` actually returns a Queryset. So I'll use `active_servers.count()` instead of `len(active_servers)`